src/MinimumNodeService.cpp

```diff
@@ -76,20 +76,9 @@ void MinimumNodeService::setUninitialised()
 //
 /// revert from Normal to Uninitialised mode
 //
-void MinimumNodeService::revertUninitialised()
+void MinimumNodeService::initSetupFromNormal()
 {
```

Comment:
Renamed as we cannot change to Uninitialized anymore.

Reply:
I was thinking of removing this function completely as it is not called by anything!  The only reason that I haven't yet is that I was not clear over if we were completely settled on not being able to change to Uninitialised.  My feeling is that we shouldn't be able to make that change other than by a factory reset.